### ISSUE TITLE
commands:NAMES Zero argument client crash

### ISSUE THREAD

**Issue by SEVENID** [Aldaviva/eircd#1]
Fix: file eircd_commands.erl line 152
```
			eircd_connector:send_server_message(User, ?IRC_NAMES_END, "#"++ChannelName++" :End of /NAMES list.");
		_ ->
			eircd_connector:send_server_message(User, ?IRC_ERROR, ":/NAMES: No channel defined.")
```
Fixed file (+ code 421 is replaced with ?IRC_ERROR): [eircd_commands.zip](https://github.com/Aldaviva/eircd/files/1200074/eircd_commands.zip)

**Comment by Aldaviva** [comment]
Hi @SEVENID,

Thanks for contributing this fix!

Can you submit your changes as a [pull request](https://help.github.com/articles/creating-a-pull-request/)?

Thanks,
Ben

**Comment by Aldaviva** [comment]
Merged Pull Request #2.
